stubs/src/Collect/Support/helpers.php

```diff
@@ -110,7 +110,7 @@ function with($object)
         function dd(...$args)
         {
             foreach ($args as $x) {
-                (new Dumper)->dump($x);
```

Comment:
Is this the correct syntax for var dumper 3, 4, and 5? Or did this syntax change between the versions?

If it changed between the versions we're going to have to diff how this runs depending on which version of the dumper they have installed.

Reply:
This syntax is for the`Illuminate\Support\Debug\Dumper` class, which was removed a while ago. 

The `symfony/var-dumper` class was mistakenly removed in probable confusion. Since the Illuminate class is no longer available and we are already using the Symfony dumper [here](https://github.com/tightenco/collect/blob/67ebd1dfc856dfc7a53197815f65483f17faa5cf/src/Collect/Support/Collection.php#L333), I updated the line within the `dd()` helper method.